src/Dan.Plugin.Arbt/Plugin.cs

```diff
@@ -1,164 +1,176 @@
+    }
 
-        var ecb = new EvidenceBuilder(_evidenceSourceMetadata, SimpleDatasetRenhold);
-        ecb.AddEvidenceValue($"Organisasjonsnummer", content.Organisasjonsnummer, SourceName);
-        ecb.AddEvidenceValue($"Status", content.Status, SourceName);
+    private async Task<List<EvidenceValue>> GetEvidenceValuesRenhold(EvidenceHarvesterRequest evidenceHarvesterRequest)
+    {
+        //The registry only has main units, so we have to traverse the enterprise structure in case the subject of the lookup is a subunit
+        var actualOrganization = await BRUtils.GetMainUnit(evidenceHarvesterRequest.OrganizationNumber, _client);
+        var url = string.Format(_settings.RenholdUrl, actualOrganization.Organisasjonsnummer);
+        var content = await MakeRequest<RenholdUnit>(url, actualOrganization.Organisasjonsnummer.ToString());
 
-        var statusChanged = Convert.ToDateTime(content.StatusEndret);
+        var ecb = new EvidenceBuilder(_metadata, "Renholdsregisteret");
+        ecb.AddEvidenceValue($"Organisasjonsnummer", content.Organisasjonsnummer, Metadata.SOURCE);
+        ecb.AddEvidenceValue($"Status", content.Status, Metadata.SOURCE);
 
-        if (statusChanged != DateTime.MinValue)
+        if (content.StatusEndret != DateTime.MinValue)
         {
-            ecb.AddEvidenceValue($"StatusEndret", statusChanged, SourceName, false);
-
+            ecb.AddEvidenceValue($"StatusEndret", content.StatusEndret, Metadata.SOURCE, false);
         }
 
         return ecb.GetEvidenceValues();
     }
+    
+    [Function("Bilpleieregisteret")]
+    public async Task<HttpResponseData> Bilpleie(
+        [HttpTrigger(AuthorizationLevel.Function, "post", Route = null)]
+            HttpRequestData req, FunctionContext context)
+    {
+        _logger = context.GetLogger(context.FunctionDefinition.Name);
+        _logger.LogInformation("Running func 'Bilpleie'");
+        var requestBody = await new StreamReader(req.Body).ReadToEndAsync();
+        var evidenceHarvesterRequest = JsonConvert.DeserializeObject<EvidenceHarvesterRequest>(requestBody);
+
+        return await EvidenceSourceResponse.CreateResponse(req, () => GetEvidenceValuesBilpleie(evidenceHarvesterRequest));
+    }
 
-    private async Task<T> MakeRequest<T>(string target)
+    private async Task<List<EvidenceValue>> GetEvidenceValuesBilpleie(EvidenceHarvesterRequest evidenceHarvesterRequest)
     {
-        HttpResponseMessage result;
+        //The registry only has main units, so we have to traverse the enterprise structure in case the subject of the lookup is a subunit
+        var actualOrganization = await BRUtils.GetMainUnit(evidenceHarvesterRequest.OrganizationNumber, _client);
+        var url = string.Format(_settings.BilpleieUrl, actualOrganization.Organisasjonsnummer);
+        var content = await MakeRequest<BilpleieUnit>(url, actualOrganization.Organisasjonsnummer.ToString());
```

Comment:
Ser Bilpleieregisteret returnerer 404 på orgnr som den ikke har, dette tryner med en EvidenceSourcePermanentClientException i MakeRequest. 

For renhold og bemanning returneres det (i stedet for å kaste) en datastruktur som indikerer at organisasjonen ikke finnes i registeret. Bemanning bør fungere på samme måte.

BRUtils.GetMainUnit vil håndtere om det er et organisasjonsnummer som ikke finnes i enhetsregisteret, og det er greit at den kaster en exception.